README.md

```diff
@@ -86,8 +86,23 @@ How to contribute to this project.
 ```
 (Step #3 creates a new branch titled <name of branch> and navigates you to that branch)
 
+## Using the API
 ### This Backend Uses JWT for authorization
 
+A JWT token is required to authenticate requests. To get a token, use the login API:
```

Comment:
Thanks for adding this! We also have the access token in a dwellinglyAccess variable in the browser's local storage when logged in, so we could describe that method here if we want as well.

![image](https://user-images.githubusercontent.com/8600792/93004615-69f83980-f4fd-11ea-97f1-ae7e90564a4f.png)